objgraph.py

```diff
@@ -582,9 +582,14 @@ def at_addrs(address_set):
     .. versionadded:: 3.4
     """
     res = []
-    for o in gc.get_objects():
-        if id(o) in address_set:
-            res.append(o)
+    id_to_obj = dict((id(o), o) for o in gc.get_objects())
+
+    for i in address_set:
```

Comment:
I suggest change these lines to 

```python
return [
    id_to_obj[i] 
    for i in address_set
    if i in id_to_obj
]

Reply:
:+1: except I'd put it all on a single line